python/ctsm/run_sys_tests.py

```diff
@@ -692,12 +696,31 @@ def _run_test_suite(
         )
 
 
+def _try_systemtests(testname_list):
```

Comment:
Cool, thanks for adding this! I see that it ended up being a little more involved than I imagined because you're checking for whether either of the relevant tests is being run... and I like that addition. Thanks!